### DISCUSSION TITLE
Persistent ECONNREFUSED Error with Supabase Cloud Database Connection

### DISCUSSION THREAD

**Discussion by immocalcul** [supabase/supabase#19231] [Questions]
Hello, we are panicked, it has been more than 24 hours since all our services have been offline and supabase cloud technical support is not keeping us informed of the situation. We have tried to update Postgres from their interface and since then nothing has happened, here is an overview of what we are encountering as if the remote server was no longer accessible. I was able to download a backup that we had. we created a new project is there a way to import this backup and restore it in a new project? in backup we only see a way to download it locally but not to import it
![supabase](https://github.com/supabase/supabase/assets/50158038/f82d00d7-13d2-43d2-ad7e-fe6cfbe8cdec)

**Comment by GaryAustin1** [comment]
Have you restarted your database and or instance?

**Comment by burggraf** [comment]
You can see  how to restore a backup here:  https://supabase.com/docs/guides/platform/migrating-and-upgrading-projects#restore-to-your-new-project
The relevant part is just using psql to run the backup script.  Something like this:

```
psql \
  --single-transaction \
  --variable ON_ERROR_STOP=1 \
  --file backup_file.sql \
  --dbname "$NEW_DB_URL"
```

Where $NEW_DB_URL is the URI full path to your database (found under "connection string in the dashboard https://supabase.com/dashboard/project/_/settings/database).
